HelpSource/Reference/randomSeed.schelp

```diff
@@ -32,8 +32,11 @@ Array.fill(7, r);
 
 subsection::Inheriting Seeds
 
-Also it is possible to set the seed of the running thread that
-all threads started within will inherit.
+When a routine is created, it "inherits" its random number generator from the parent thread (link::Classes/Thread#.thisThread#thisThread::). If no particular randSeed is set (the normal case), it will tap the same stream of random number generators used by the parent.
+
+The parent thread is whichever Routine is in force at the moment of creating a new Routine. If no Routine is evaluating at that moment, then the parent is the sclang process master link::Classes/Thread:: -- that is, the default from which all Routines inherit the random number generator.
```

Comment:
s/master/main

Reply:
this last clause is a bit oddly worded -- consider "in other words, by default all Routines inherit their RNGs from the main thread". or just nixing it.